### PR TITLE
Nessus_pro_support_doc

### PR DESCRIPTION
**Pull request by VatsalJagani**

### CODE REVIEW

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 1]
```diff
@@ -6180,6 +6180,7 @@ display.page.search.tab = statistics
 display.page.search.mode = fast
 request.ui_dispatch_app = cyences_app_for_splunk
 request.ui_dispatch_view = search
+# TODO - need to add missing fields for this search to props.conf for Nessus:Pro
```
Comment (mahirchavda) [inline cyences_app_for_splunk/default/savedsearches.conf:6183]: I think all fields are added. can we remove this

Comment (VatsalJagani) [reply cyences_app_for_splunk/default/savedsearches.conf:6183]: This don't need in the DOC PR anyways.

**Comment by mahirchavda** [comment]
FYI, The PR still has many code changes.

**Comment by VatsalJagani** [comment]
> FYI, The PR still has many code changes.

Yeah bro, I noticed and tried to remove it. But re-add it as this will create merge conflict in the future. So keeping it same as other branch is safe option to remember in the future on what code is right :) ha ha ha!!

**Review by hardikhdholariya — APPROVED** [review]

**Pull request merged**